src/Apps/Search/Components/ZeroState.tsx

```diff
@@ -1,14 +1,21 @@
 import { Box, Flex, Serif } from "@artsy/palette"
 import { SendFeedback } from "Apps/Search/Components/SendFeedback"
+import qs from "qs"
 import React, { FC } from "react"
 
 interface Props {
   term: string
-  entity: string
 }
 
 export const ZeroState: FC<Props> = props => {
-  const { term, entity } = props
+  const { term } = props
+  let hasFilters = false
+  if (window.location.search.length) {
+    const params = Object.keys(
+      qs.parse(window.location.search, { ignoreQueryPrefix: true })
+    )
+    hasFilters = params.length > 1
+  }
```

Comment:
Can this be pulled from the router?

```
this.router.props.search
```

In general (with a few exceptions) all router / history state needs to go through [found](https://github.com/4Catalyzer/found) / [farce](https://github.com/4Catalyzer/farce) so that state is not out of sync.

Reply:
This is a case where we are manually pushing things to window.history, via [the FilterState state container](https://github.com/artsy/reaction/blob/76408fc817cc79aea707e9d75076c20531584889/src/Apps/Search/FilterState.tsx#L132); I don't think any of these URL updates are currently going through found/farce.

I _think_ the reason we are manually pushing things to the history instead of navigating via a found route is that we want the user to feel like they aren't leaving this page, they are just getting different results filtered. Pushing them to a new route would, I would expect, be more of a jarring experience than we want them to feel. I could be wrong about that; I don't know the history, and I'm mostly guessing, and I also don't know if found is capable of doing a transition between routes without re-rendering the entire route.

Reply:
There are a couple things that come to mind for modifying the code in this PR to not check `window.location`.

1. _Can_ we use found to manage the routes that are changed when you interact with the filters?
2. We could elevate the `FilterState` provider above this component in the tree. I _think_ the hierarchy currently looks like this:

```
<SearchApp>
  <ZeroState/>
  <SearchResultsArtworksRoute>
    <Provider inject={new FilterState(...)}>
      <SearchResultsFilterContainer>....actual filters and search results...</SearchResultsFilterContainer>
    </Provider>
  </SearchResultsArtworksRoute>
</SearchApp>
```

And I think we'd want to change that to be more like this:

```
<SearchApp>
  <Provider inject={new FilterState(...)}>
    <ZeroState/>
    <SearchResultsArtworksRoute>
      <SearchResultsFilterContainer>....actual filters and search results...</SearchResultsFilterContainer>
    </SearchResultsArtworksRoute>
  </Provider>
</SearchApp>
```

And then the `ZeroState` component would have access to the current filters in the `FilterState`.

Reply:
Thanks @pepopowitz for looking into this, at this point if we need to refactor something just for this copy change its probably not worth it. The question is, I think, if we're capturing the location state correctly as is.